Editor/Window/GameLiftRequestAdapter.cs

```diff
@@ -0,0 +1,242 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Amazon.GameLift;
+using Amazon.GameLift.Model;
+using AmazonGameLift.Editor;
+using UnityEngine;
+using UnityEngine.UIElements;
+
+namespace Editor.Window
+{
+    public class GameLiftRequestAdapter
+    {
+        private GameLiftPlugin _gameLiftConfig;
```

Comment:
minor: Consider renaming the variable to _gameliftPlugin for clarity.

Reply:
renamed